common/app/controllers/EmailSignupController.scala

```diff
@@ -78,23 +78,19 @@ class EmailFormService(wsClient: WSClient, emailEmbedAgent: NewsletterSignupAgen
         .obj(
           "email" -> form.email,
           "set-lists" -> form.listNames,
+          "refViewId" -> form.refViewId,
```

Comment:
Why are we sending these now? Does IDAPI consume them?

Reply:
Oh I see, they were in the query string before, but now they're in the body?

Reply:
Yes. I just moved these to the body.